deployer/scripts/upgrade.sh

```diff
@@ -731,6 +731,7 @@ function update_for_common_data_model() {
   count=$(get_list_of_proj_uuid_indices | wc -l)
   if [ $count -eq 0 ] ; then
       echo No matching indexes found - skipping update_for_common_data_model
+      rm -f $indices
```

Comment:
I don't know if any of this is needed. If customers upgrade to the latest 1.4 before upgrading to 1.5, this won't be necessary.  If customers are upgrading from 1.3 -> 1.5, I have no idea if that will work.

At any rate, I don't think any of this code is used anymore in 1.5.